scripts/babel-preset-emotion-dev/src/index.js

```diff
@@ -11,8 +11,10 @@ module.exports = () => {
       require.resolve('babel-plugin-fix-dce-for-classes-with-statics'),
       require.resolve('@babel/plugin-transform-flow-strip-types'),
       require.resolve('babel-plugin-codegen'),
-      require.resolve('@babel/plugin-proposal-object-rest-spread'),
-      require.resolve('@babel/plugin-transform-runtime'),
+      [
+        require.resolve('@babel/plugin-transform-runtime'),
+        { version: require('@babel/runtime/package.json').version }
```

Comment:
this allows inserting references to `@babel/runtime` for helpers which were not available yet in 7.0.0-beta.0 release